modules/basics/src/main/java/com/opengamma/strata/basics/market/MarketData.java

```diff
@@ -60,12 +61,12 @@ public static MarketData of(
 
   //-------------------------------------------------------------------------
   /**
-   * Checks if this set of data contains a value for the specified key.
+   * Returns a box containing values for the specified ID if available.
```

Comment:
That's a copy-paste error from the version of `findValue` returning a `MarketDataBox`